src/common/json_output.rs

```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 #![deny(missing_docs)]
 #![deny(warnings)]
+#![allow(clippy::too_many_arguments)]
```

Comment:
why  is this needed ?

Reply:
new() function for EnclaveDescribeInfo was taking 7 arguments (the limit before clippy throws an error for too many arguments) so adding measurements would surpass that. I didn't see the need of refactoring to reduce the fields of the structure as they are all part of the info we need.

Reply:
ack